src/main/java/core/basesyntax/services/impl/ValidatorImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.services.impl;
+
+import core.basesyntax.exceptions.ValidatorException;
+import core.basesyntax.services.Validator;
+import java.util.List;
+import java.util.regex.Pattern;
+
+public class ValidatorImpl implements Validator {
+    private static final String INPUT_TITLE = "type,fruit,quantity";
+    private static final String PATTERN_LINE = "[bprs],[a-z]*,[0-9]*";
+
+    @Override
+    public void validate(List<String> line) {
+        if (line.isEmpty()) {
```

Comment:
```suggestion
        if (line.isBlank()) {
```

Reply:
it's not necessary to change it, isEmpty - is a method, and informative quite enough;

Reply:
if we have line = " " ?;

Reply:
it's not an evaluation for a string - but for list

Reply:
okey